packages/app/universal-ts-utils/src/array/areStringArraysEqual.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Returns true if the two arrays are equal, false otherwise.
+ * The arrays are considered equal if they have the same length, same order and the elements at each index are equal.
+ */
+export function areStringArraysEqual(a: string[], b: string[]): boolean {
```

Comment:
🟢 Nit: Wondering if would be useful to extend it to be used with other basic types (eg: numbers)